lace/emulator/p1d_archive.py

```diff
@@ -131,7 +141,8 @@ def _load_data(self,drop_tau_rescalings,drop_temp_rescalings,
                 # setup CAMB object
                 sim_cosmo=camb_cosmo.get_cosmology_from_dictionary(sim_cosmo_dict)
                 # compute linear power parameters at each z (in Mpc units)
-                linP_zs=fit_linP.get_linP_Mpc_zs(sim_cosmo,zs,self.kp_Mpc)
+                linP_zs=fit_linP.get_linP_Mpc_zs(sim_cosmo,zs,self.kp_Mpc,
+                        include_f_p=True)
```

Comment:
Why did you need to specify include_f_p=True? I thought this was obsolete code. Maybe you merged from an older version of master?

Reply:
I did not modify any of this, and I think I started from the latest master version, buI may be wrong...

Reply:
I think I cleaned up these old pieces of code (include_f_p) a few months ago. I'll take a look at it myself, but I'm pretty sure we can go back to removing include_f_p.